components/create_comment/create_comment.jsx

```diff
@@ -444,7 +444,9 @@ export default class CreateComment extends React.PureComponent {
 
         if (allowSending) {
             e.persist();
-            this.refs.textbox.blur();
+            if (this.refs.textbox) {
+                this.refs.textbox.getWrappedInstance().blur();
```

Comment:
I'd love feedback on connecting `Textbox` and updating the world to poke through the wrapped instance like this. An alternative was to push up the the store access to all components that used the `Textbox`, but that seems like a violation of concerns. Alternatively, I could have accessed the store globally from within the suggestion provider, but there's currently no way to "trigger" the user of the provider to try a pretext match.

Reply:
I'm not sure I understand. Do you just mean to avoid the mess of `getWrappedInstance`?

Reply:
Yeah, that :/